src/main/js/components/Staff.js

```diff
@@ -3,6 +3,7 @@ import React from "react";
 import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
 
 /** ----- COMPONENT IMPORTS -----**/
+import {CustomerDiningSessionSelect, StaffDiningSessionPage, StaffDiningSession} from "./subcomponents/DiningSession";
```

Comment:
What's the purpose of importing `CustomerDiningSessionSelect` as well as `StaffDiningSession` here?